src/sentry/static/sentry/app/api.tsx

```diff
@@ -442,7 +438,19 @@ export class Client {
           // No text came out.. too bad
         }
 
-        const {ok, status, statusText} = response;
+        if (status !== 204 && status !== 304) {
```

Comment:
Should this be `status < 204 || status >= 400`? We likely use 302 in addition to 304.

Reply:
Would `status != 204 && status < 300 && status >= 400` be better? This way it's only skipping 204 and 3xx codes. There's the possibility that we use other 2xx codes greater than 204.

And fwiw, these two status codes were lifted from https://github.com/jquery/jquery/blob/main/src/ajax.js#L774-L788

Reply:
That works for me :+1: